editor/gui/scene_tree_editor.h

```diff
@@ -109,6 +110,7 @@ class SceneTreeEditor : public Control {
 	bool tree_dirty = true;
 	bool pending_test_update = false;
 	static void _bind_methods();
+	Timer *update_node_tooltip_delay;
```

Comment:
```suggestion
	Timer *update_node_tooltip_delay = nullptr;
```
Initialize non-simple types

Reply:
Thanks!